mozilla/pontoon
# Fix bug 1590020 - Disable allCapitalString parser

#### What does this PR do?
This pull request fixes the issue [1590020](https://bugzilla.mozilla.org/show_bug.cgi?id=1590020). It has allCapitalsString parser removed entirely from the codebase. It ensures all capital words are not matched anymore in the UI and all tests pass.

#### Type of change:

#### How Has This Been Tested?
- Clone this repository 
- Git checkout to `katherine95:1590020-disable-allCapitalString-parser` branch.
- Follow the setup guidelines to set up pontoon locally
- Run `make build` then `make run`
- Login to Pontoon
- Navigate to the playground and add a translation in capital letters.
`http://localhost:8000/projects/tutorial/playground/`
- The translation in capital letters should not be highlighted or matched.

#### Checklist:
- [x]The translation in capital letters should not be highlighted or matched.

#### Screenshots (if appropriate):
Before
![photo5863987423132889444](https://user-images.githubusercontent.com/17095461/67407589-69db0c80-f5c0-11e9-99e7-055c67d9923d.jpg)

After
![Screenshot from 2019-10-23 18-00-40](https://user-images.githubusercontent.com/17095461/67407605-6f385700-f5c0-11e9-98b9-75b6bc59f7d7.png)
